File: Iniciante/URI 1006.cs
using System;

class URI {

    static void Main(string[] args) {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double media = ((a * 2) + (b * 3) + (c * 5)) / 10;
        Console.WriteLine("MEDIA = {0:F1}", media);
    }
    
}

File: Iniciante/URI 1006.dart
import 'dart:io';

void main() {
  double a = double.parse(stdin.readLineSync());
  double b = double.parse(stdin.readLineSync());
  double c = double.parse(stdin.readLineSync());
  double media = ((a * 2) + (b * 3) + (c * 5)) / 10;
  print("MEDIA = ${media.toStringAsFixed(1)}");
}